README.md

```diff
@@ -5,6 +5,40 @@
 # 2022-software-engineering-projects-pk
 By Isaac Ellmen (Ellmen), Leonard Lee (leonard-lenny-lee) & Raman van Wee (Ramanspec). Project development ceased on 21 Oct 2022
 
+## Installation
+
+```
+pip install -r requirements.txt
+pip install -e .
+```
+
+## Usage
+```
+from pkmodel import Compartment, Dose, SpikeDose, Model, Protocol, Solution
+
+
+c1 = Compartment(c_type="peripheral", rate=1.0, volume=1.0)
+c2 = Compartment(c_type="peripheral", rate=2.0, volume=1.0)
+
+
+d1 = Dose(rate=6, start=0, end=1)
+d2 = SpikeDose(volume=4, start=0)
+d3 = SpikeDose(volume=2, start=0.5)
+p1 = Protocol(dosing_strategy=[d1])
+p2 = Protocol(dosing_strategy=[d2,d3])
+
+m1 = Model(name='model1', volume=1.0, clearance_rate=1.0, peripherals=[c1], protocol=p1)
+m2 = Model(name='model2', volume=1.0, clearance_rate=1.0, peripherals=[c1, c2], protocol=p2)
+
+models = [m1, m2]
+s = Solution(models=models, therapeutic_min=1, therapeutic_max=3)
+
+s.solve()
+s.plot()
+# s.save_plot()
```

Comment:
@Ramanspec this can be uncommented in the README. Also, we should update this soon with an example with a dosing compartment once https://github.com/Ellmen/pharmokinetics/pull/27 gets merged.